vllm/worker/vineyard_llm_cache.py

```diff
@@ -360,10 +424,16 @@ def update_seq_kv_caches(
             ) = update_args
         if update_token_size <= 0:
             # restore the seq_group_metadata's and seq's metadata
-            if seq_group_metadata is not None:
-                seq_data.update_num_computed_tokens(-matched[seq_id])
-                seq_group_metadata.token_chunk_size += matched[seq_id]
-            return seq_id, 0
+            self._update_seq_group_metadata(seq_group_metadata, -matched[seq_id])
+            return
+
+        buffer_tensors_tuple = self.tensor_pool.get(block=False)
+        if buffer_tensors_tuple is None:
```

Comment:
what does `buffer_tensors_tuple is None` mean here? there's no available object which indicates it reaches the maximum in-flight requests?

Reply:
yes, will add a comment here

Reply:
added